src/test/java/org/tikv/common/PDClientTest.java

```diff
@@ -68,6 +68,15 @@ public void testTso() throws Exception {
     }
   }
 
+  @Test
+  public void testPauseCheck() throws Exception {
+    try (PDClient client = session.getPDClient()) {
+      client.keepPauseChecker(PDChecker.Merge);
+      Thread.sleep(6000);
```

Comment:
can we get the current status (pause or not)  and add assert here?

Reply:
Ok. `isCheckerPaused` is added.